### ISSUE TITLE
the code about training clip merging model

### ISSUE THREAD

**Issue by NSN-Hello** [tberriel/OVO#7]
Would the the code about training clip merging model  be released ？

**Comment by tberriel** [comment]
Hi, sorry for the late response!
We do not plan to release the training code nor preprocessed data. If you want to reproduce the training setup, most of the details are provided in the [last version of the paper](https://arxiv.org/abs/2411.15043).

Regarding training data, we used ScanNet++v1 training set of 250 scenes, most of them can be found in ScanNet++v2. We preprocessed projecting 3D instances to 2D to semantic ground-truth classes. Then we used SAM to generate 2D segmentation masks and labeled them with the mode of ground-truth classes inside each mask. Finally, we used these masks and SigLip to compute the three input descriptors as described in the paper, and these were used during training.

Best regards
Tomás
